composer.json

```
@@ -11,31 +11,17 @@
   ],
   "config": {
     "platform": {
-      "php": "5.3.3"
+      "php": "8.1"
     },
     "vendor-dir": "src/vendors"
   },
-  "repositories": [
-    {
-      "type":"package",
-      "package": {
-        "name": "shopgate/prestashop-backward-compatibility",
-        "version":"1.0",
-        "source": {
-          "url": "https://github.com/PrestaShop/PrestaShop-backward_compatibility",
-          "type": "git",
-          "reference":"master"
-        }
-      }
-    }
-  ],
   "require": {
-    "php": ">=5.3",
-    "shopgate/cart-integration-sdk" : "~2.9.71",
-    "shopgate/prestashop-backward-compatibility": "1.0"
+    "php": ">=8.1",
+    "shopgate/cart-integration-sdk" : "2.10.2",
```

```suggestion
    "shopgate/cart-integration-sdk" : "^2.10.3",
```

+ remove composer.lock file.